doc/start/os-recommendations.rst

```diff
@@ -41,6 +41,42 @@ platforms.  Generally speaking, there is very little dependence on
 specific distributions aside from the kernel and system initialization
 package (i.e., sysvinit, systemd).
 
++--------------+--------+--------+--------+----------+--------+
+| Release Name | Tag    | CentOS | Ubuntu | OpenSUSE | Debian |
++==============+========+========+========+==========+========+
+| Quincy       | 17.2.z | 8      | 20.04  | 15.3     | 10     |
++--------------+--------+--------+--------+----------+--------+
+| Pacific      | 8      | 8      | 20.04  | 15.2     | 10     |
++--------------+--------+--------+--------+----------+--------+
+| Octopus      | 20.04  | 8      | 18.04  | 15.2     | 10     |
++--------------+--------+--------+--------+----------+--------+
```

Comment:
debian should be 11 for quincy, and 10 and 11 for pacific like david mentioned. For pacific and octopus we have ubuntu 18.04 and 20.04, and for just octopus, centos 7 and 8..

Reply:
I have included this information in this table.